module/zfs/dsl_scan.c

```diff
@@ -1686,6 +1756,12 @@ dsl_scan_sync(dsl_pool_t *dp, dmu_tx_t *tx)
 		return;
 	}
 
+	if (dsl_scan_is_paused_scrub(scn)) {
+		zfs_dbgmsg("bailing on scrub since it's paused, sync txg %llu",
+		    (longlong_t)tx->tx_txg);
```

Comment:
This message might be a bit excessive, since it will be printed every txg, even though we aren't doing anything.  My general philosophy around zfs_dbgmsg is that we should use it when something exceptional happens, when we make roughly once-per-txg on-disk changes, and roughly once-per-txg when we make progress on background tasks.  But simply saying "the system continues to be in this normal state" every txg seems like a waste of log space.